doc/en/components/inputs/file-input.md

```diff
@@ -0,0 +1,165 @@
+---
+title: {Platform} File Input | Data Visualization Tools | Infragistics
+_description: Infragistics' {Platform} File input is a component where the user can select and upload files. Improve your application with {ProductName}!
+_keywords: {Platform} File input, {ProductName}, Infragistics
+mentionedTypes: ["Input", "Icon", "Button"]
+---
+# {Platform} File Input Overview
+
+The {ProductName} File Input component provides an interactive way for users to select and upload files. It extends the base `Input` functionality by adding file-specific features such as file selection, displaying selected file names, and supporting multiple file uploads.
+
+## {Platform} File Input Example
+
+<div class="divider--half"></div>
+
+`sample="/inputs/file-input/overview", height="120", alt="{Platform} File Input Overview Example"`
+
+## Usage
+
+The File Input component allows users to select files from their device and upload them to a web application. It displays the names of selected files and offers customization options for the browse button and the "No file chosen" text. The component also provides properties, methods, and slots that can be used to further configure its behavior to suit your needs.
+
+### Getting Started
+
+<!-- WebComponents -->
+
+To start using the File Input, first, you need to install the Ignite UI for Web Components by running the following command:
+
+```cmd
+npm install {PackageWebComponents}
+```
+
+After that, you need to import the File Input as follows:
+
+```ts
+import { defineComponents, IgcFileInputComponent } from 'igniteui-webcomponents';
+
+defineComponents(IgcFileInputComponent);
+```
+
+Now you can start with a basic configuration of the {Platform} File Input.
+
+```html
+  <igc-file-input label="File Input" required=true></igc-file-input>
+```
+
+For a complete introduction to the {ProductName}, read the [*Getting Started*](../general-getting-started.md) topic.
+
+## Configuration
+
+### Properties
+
+The File Input component offers a variety of properties that allow you to configure its behavior based on specific requirements. These properties give you control over the input’s functionality, appearance, and validation.
+
+* `value` - Sets the current value of the file input field.
+* `disabled` -	Disables the file input, preventing user interaction.
+* `required` -	Marks the input as mandatory. Form submission will be blocked unless a file is selected.
+* `invalid` -	Indicates that the input value is invalid, used to trigger visual error states.
+* `multiple` -	Allows the selection of multiple files.
+* `accept` -	Defines the types of files that can be selected. The value for this property needs to be a comma-separated list of file formats (e.g., .jpg, .png, .gif).
+* `autofocus` -	Automatically focuses the file input field when the page loads.
+* `label` -	Sets the label text associated with the file input element.
+* `placeholder` -	Provides placeholder text displayed when no file is selected.
+
```

Comment:
The formatting of this is all over the place. Please fix it. Make sure that the spaces between the dashes and the text is the same for all items.